library/core/src/num/f64.rs

```diff
@@ -440,6 +440,32 @@ impl f64 {
         self.abs_private() < Self::INFINITY
     }
 
+    /// Returns `true` if the number is [subnormal].
+    ///
+    /// ```
+    /// #![feature(is_subnormal)]
+    /// let min = f64::MIN_POSITIVE; // 1.17549435e-38f64
```

Comment:
```suggestion
    /// let min = f64::MIN_POSITIVE; // 2.2250738585072014e-308_f64
```

Reply:
I feel silly for not even thinking of this. 🤦🏻